app/cdap/components/PipelineTriggers/EnabledTriggersTab/EnabledTriggerRow.tsx

```diff
@@ -142,6 +143,13 @@ const EnabledTriggerRowView = ({
           {info ? <span>{info && info.description}</span> : renderLoading()}
           <PipelineLink
             href={`/pipelines/ns/${triggeringPipelineInfo.namespace}/view/${triggeringPipelineInfo.id}`}
+            target="_blank"
+            // The Anchor tab is not working, using this hacky way to fix it for now
```

Comment:
what do you mean anchor tag is not working?

Reply:
oh, this should be removed

Reply:
I meant removing the href and use the window.open workaround only